qemu/tests/rh_qemu_update.py

```diff
@@ -0,0 +1,302 @@
+import requests
+import json
+import re
+
+from avocado.utils import process
+from virttest import error_context
+
+QUERY_TIMEOUT = 360
+INSTALL_TIMEOUT = 360
+OPERATION_TIMEOUT = 7200
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test update of qemu-kvm
+    1) Boot the VM
+    2) Install qemu-kvm metapackage from compose
+    3) Clone component_management tool
+    4) Verify available, host and guest qemu versions
+    5) Update qemu-kvm using component_management
+    6) Verify installed qemu-kvm version
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment
+    """
+    qemu_package = params['qemu_package_install']
+
+    def install_qemu(session):
+        """
+        Install compose version of qemu-kvm pkg by the name provided in cfg
+        """
+        cmd = "{} install -y {}".format(
+            params['pm_tool'], qemu_package
+        )
+        s, o = session.cmd_status_output(cmd, timeout=OPERATION_TIMEOUT)
+        if s != 0:
+            test.error("Installation of '{}' failed with: {}".format(
+                qemu_package, o
+                )
+            )
+        test.log.info("Installation of '{}' succeeded".format(
+            qemu_package
+            )
+        )
+
+    def install_component_management(session):
+        """
+        Clone component management repository for preparation of necessary
+        repositories containing qemu-kvm build or whole virt module
+        """
+        cmd_clone = "git clone --depth=1 {} -b {} {}".format(
+            params['cm_repo'],
+            params['cm_branch'],
+            params['cm_path']
+        )
+        s_clone, o_clone = session.cmd_status_output(
+            cmd_clone, timeout=OPERATION_TIMEOUT
+            )
+        if s_clone != 0:
+            test.error("Clonning of '{}' failed with: {}".format(
+                params['cm_repo'], o_clone
+                )
+            )
+        test.log.info("Clonning of '{}' succeeded".format(
+            params['cm_repo']
+            )
+        )
+
+    def _compare_qemu_versions(nvr_1, nvr_2):
```

Comment:
`LooseVersion` can easily implement this function.
ex:
```
>>> from distutils.version import LooseVersion
>>> LooseVersion('qemu-kvm-6.2.0-0.el9') < LooseVersion('qemu-kvm-6.2.0-0.el9_0.1')
True
>>> LooseVersion('qemu-kvm-6.2.0-0.el9_0.1') < LooseVersion('qemu-kvm-6.2.0-0.el9_0.1')
False
>>> LooseVersion('qemu-kvm-6.2.0-0.el9_0.2') < LooseVersion('qemu-kvm-6.2.0-0.el9_0.1')
False
>>> LooseVersion('qemu-kvm-6.2.0-1.el9') < LooseVersion('qemu-kvm-6.2.0-0.el9_0.1')
False
```

Reply:
Ah, this one is very nice! It will simplify things a lot.

Reply:
Done. I could drop a lot of code, thanks!